### ISSUE TITLE
Cyrillic support (Russian translation)

### ISSUE THREAD

**Issue by StivenFox** [aa2g/AA2Unlimited#68]
Greetings,
To begin with, I want to say that the program is simply wonderful, it introduces so much variety, it's a pity I learned about it not so long ago.
And now the actual question is whether there is an opportunity to make sure that the font of the Cyrillic alphabet is displayed correctly in the game.
For example, a couple of screenshots of how it looks at the moment.
https://imgur.com/a/tmltD
Once upon a time a craftsman made Launcher supporting the Russian translation, but he has no special desire to update it, and the files themselves are stitched, I do not know whether this will be useful, but I will send them to you. If necessary, I can ask not encrypted, unless of course he will agree.

Here are links to the translation part files and launchers.
https://mega.nz/#!ZdwC0TgZ!BNI77eQ4_1fx3cCGiaGk_kv31_faTnWVUsfMYcZsN8g
https://mega.nz/#!9JpRSLzb!cg7jooph_VFXHrOm__SdZmaijEv-e4g5YS-658FuGaM

And another question is not so important, but I'm interesting, can I change the font in the game? It would be nice to make it a script, changing one single file in the folder with the program.

I look forward to hearing from you.

**Comment by linkanon** [comment]
Here, I've got a modified MKIII launcher. Haven't seen a SVII version so you might get a case of disco-penis.
[AA2Play v12 rus MKII - AAU.zip](https://github.com/aa2g/AA2Unlimited/files/1687766/AA2Play.v12.rus.MKII.-.AAU.zip)

It's prepatched with wrong directory though but works with the standard patching procedure(put next to your `AA2Play.exe` and run the `AAUnlimited\install.bat`)

Credit belongs to the guy that made it

**Comment by StivenFox** [comment]
Yes, I was already given an answer to my question in the discord, though I did not expect it would be so fast. Thank you.)))

**Comment by linkanon** [comment]
Reopened in case of future inquiries
